profiles/seccomp/default_linux.go

```diff
@@ -225,6 +225,9 @@ func DefaultProfile() *Seccomp {
 				"openat",
 				"openat2",
 				"pause",
+				"pidfd_getfd",
```

Comment:
Should this be toggled with CAP_SYS_PTRACE as in process_vm_readv and process_vm_writev?

Reply:
I think yes. Manual page mentions about ptrace permissions check.

Reply:
Suggested changes have been made.